src/main/java/core/dao/car/CarDaoJdbc.java

```diff
@@ -0,0 +1,207 @@
+            getAllStatement.setLong(1, driverId);
+            ResultSet resultSet = getAllStatement.executeQuery();
+            while (resultSet.next()) {
+                Car car = DaoUtils.parseToCar(resultSet);
+                car.setDriverList(getDrivers(car, connection));
+                returnList.add(car);
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException(String
+                    .format(GET_CARS_BY_DRIVER_EXCEPTION, driverId), e);
+        }
+        return returnList;
+    }
+    
+    private void insertDrivers(Car car, Connection connection) {
+        String insert = "INSERT INTO cars_drivers(\"car_Id\", \"driver_Id\") VALUES ("
+                        + car.getId() + ", ?);";
+        try (PreparedStatement insertStatement = connection.prepareStatement(insert)) {
+            for (Driver driver : car.getDriverList()) {
+                insertStatement.setLong(1, driver.getId());
+                int wasAdded = insertStatement.executeUpdate();
+                if (wasAdded == 0) {
+                    throw new DataProcessingException(String.format(INSERT_DRIVER_EXCEPTION,
+                            driver.getId()));
+                }
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException(String.format(INSERT_DRIVER_EXCEPTION,
+                    car.getId()), e);
+        }
+    }
+    
+    private void removeDrivers(Car car, Connection connection) {
+        String remove = "DELETE FROM cars_drivers WHERE \"car_Id\" = " + car.getId()
+                + " AND EXISTS(SELECT id WHERE \"car_Id\" = ?)";
+        try (PreparedStatement removeStatement = connection.prepareStatement(remove)) {
+            removeStatement.setLong(1, car.getId());
+            removeStatement.executeUpdate();
+        } catch (SQLException e) {
+            throw new DataProcessingException(String
+                    .format(REMOVE_DRIVER_EXCEPTION, car.getId()), e);
+        }
+    }
+    
+    private List<Driver> getDrivers(Car car, Connection connection) {
+        String select = "SELECT DISTINCT drivers.id, drivers.name, drivers.licence_number"
+                        + " FROM drivers"
+                        + " INNER JOIN cars_drivers cd ON drivers.id = cd.\"driver_Id\""
+                        + " WHERE deleted = false AND cd.\"car_Id\" = " + car.getId();
+        List<Driver> list = new ArrayList<>();
+        try (PreparedStatement selectDrivers = connection.prepareStatement(select)) {
+            ResultSet resultSet = selectDrivers.executeQuery();
+            while (resultSet.next()) {
+                list.add(DaoUtils.parseToDriver(resultSet));
+            }
+        } catch (SQLException e) {
+            throw new DataProcessingException(String
+                    .format(GET_DRIVERS_EXCEPTION, car.getId()), e);
+        }
+        return list;
+    }
```

Comment:
You can replace this method to DaoUtils and invoke only 1 time in `parseToCar()`

Reply:
I use methods of this class in DriverDao, CarDao, ManufacturerDao specifically to avoid code duplication